packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -388,30 +388,66 @@ export class MirrorNodeClient {
+        throw error;
+    }
+
     public async getAccount(idOrAliasOrEvmAddress: string, requestIdPrefix?: string) {
         return this.get(`${MirrorNodeClient.GET_ACCOUNTS_BY_ID_ENDPOINT}${idOrAliasOrEvmAddress}?transactions=false`,
             MirrorNodeClient.GET_ACCOUNTS_BY_ID_ENDPOINT,
             requestIdPrefix);
     }
 
-    public async getAccountLatestEthereumTransactionsByTimestamp(idOrAliasOrEvmAddress: string, timestampTo: string, numberOfTransactions: number = 1, requestIdPrefix?: string) {
+    /**
+     * Returns account or null if account does not exist
+     * @param {string} idOrAliasOrEvmAddress commonly an evm address
+     * @param {string} requestIdPrefix optional request id prefix
+     */
+    public async getAccountOrNull(idOrAliasOrEvmAddress: string, requestIdPrefix?: string) {
+        try {
+            return await this.getAccount(idOrAliasOrEvmAddress, requestIdPrefix);
+        } catch (error: any) {
+            return this.handleAccountNotFound(error, idOrAliasOrEvmAddress, requestIdPrefix);
+        }
+    }
+
+    public async getAccountLatestEthereumTransactionsByTimestampOrNull(idOrAliasOrEvmAddress: string, timestampTo: string, numberOfTransactions: number = 1, requestIdPrefix?: string) {
         const queryParamObject = {};
         this.setQueryParam(queryParamObject, MirrorNodeClient.ACCOUNT_TRANSACTION_TYPE_PROPERTY, MirrorNodeClient.ETHEREUM_TRANSACTION_TYPE);
         this.setQueryParam(queryParamObject, MirrorNodeClient.ACCOUNT_TIMESTAMP_PROPERTY, `lte:${timestampTo}`);
         this.setLimitOrderParams(queryParamObject, this.getLimitOrderQueryParam(numberOfTransactions, constants.ORDER.DESC)); // get latest 2 transactions to infer for single case
         const queryParams = this.getQueryParams(queryParamObject);
 
-        return this.get(
-            `${MirrorNodeClient.GET_ACCOUNTS_BY_ID_ENDPOINT}${idOrAliasOrEvmAddress}${queryParams}`,
-            MirrorNodeClient.GET_ACCOUNTS_BY_ID_ENDPOINT,
-            requestIdPrefix
-        );
+        try {
+            return await this.get(
+                `${MirrorNodeClient.GET_ACCOUNTS_BY_ID_ENDPOINT}${idOrAliasOrEvmAddress}${queryParams}`,
+                MirrorNodeClient.GET_ACCOUNTS_BY_ID_ENDPOINT,
+                requestIdPrefix
+            );
+        } catch (error: any) {
+            return this.handleAccountNotFound(error, idOrAliasOrEvmAddress, requestIdPrefix);
+        }
     }
 
-    public async getAccountPageLimit(idOrAliasOrEvmAddress: string, requestIdPrefix?: string) {
-        return this.get(`${MirrorNodeClient.GET_ACCOUNTS_BY_ID_ENDPOINT}${idOrAliasOrEvmAddress}?limit=${constants.MIRROR_NODE_QUERY_LIMIT}`,
-            MirrorNodeClient.GET_ACCOUNTS_BY_ID_ENDPOINT,
-            requestIdPrefix);
+    public async getAccountPageLimitOrNull(idOrAliasOrEvmAddress: string, requestIdPrefix?: string) {
```

Comment:
Q: why the `OrNull` method overload?
Should services calling this handle the null?

Reply:
Same as above